flow-editor/src/visual-node-editor/VisualNodeEditor.tsx

```diff
@@ -765,14 +782,16 @@ export const VisualNodeEditor: React.FC<VisualNodeEditorProps & { ref?: any }> =
 
       const selectAll = React.useCallback(() => {
         const allIds = node.instances.map((i) => i.id);
-        onChangeBoardData({ selected: allIds, from: undefined, to: undefined });
+        onChangeBoardData({ selected: allIds, selectedConnections: [], from: undefined, to: undefined });
       }, [onChangeBoardData, node.instances]);
 
       const onDeleteInstances = React.useCallback(
         (ids: string[]) => {
           const newConnections = connections.filter(({ from, to }) => {
             return (
-              ids.indexOf(from.insId) === -1 && ids.indexOf(to.insId) === -1
+              !ids.includes(getConnectionId({from, to})) &&
+              ids.indexOf(from.insId) === -1 &&
+              ids.indexOf(to.insId) === -1
```

Comment:
my bad, but.. boy scout rule
```suggestion
              !ids.includes(from.insId) &&
              !ids.includes(to.insId)
```